generate.sh

```diff
@@ -4,9 +4,12 @@ set -ex
 
 DSTEP=$1
 SRCDIR=$2
-LIBSODIUMD=$3/source/libsodium/
+LIBSODIUMD=$3/source/libsodium
 export LIBSODIUMD
 
+# Remove all previous files
+rm -rf ${LIBSODIUMD}/*
```

Comment:
This commit ("Previous generated files removal added") should be squashed with the commit that adds the script